### DISCUSSION TITLE
Gamerized Discussion

### DISCUSSION THREAD

**Discussion by Heiwa1580** [Glimesh/company#25] [Business Partnerships]
Hi All, 

I spoke with [Gamerized](https://gamerized.com/) yesterday in terms of starting the discussion around partnership. Please take some time and explore their site and what they do before posting thoughts. Thanks all would love to get some thoughts.

**This discussion is in development, and could change at any time**

Highlights

- They want to more centralized platform for streamers and gamers of all kinds. 
- The goal is to have one area for all their content.
- They would like to see if we could integrate into their service in a way where live streams would show up in a live stream section and when people clicked on those it would direct them to Glimesh but be viewable on Gamerized. Basically so that they can involve live streaming on their platform without needing to create it themselves. We would give them the ability to have streams on their site without them needing to directly run streaming.
- They would basically offer their services as well, they provide artists a platform to upload DMCA safe music and for streamers to use that music. They also offer a video host section like Youtube. And they offer a social network experience.
- They also offered that if the streams were on their service that they would be willing to host all the VODs associated with the streams on Gamerized. 
- Then there would be some aspects of co-advertising. They would push people to the platform using their platform

They would like us

- To assist in the establishment of getting live streams on their site. 
- Co-advertising through social media
- Sharing their DMCA free music on the page
- Placement on a partnership page or similar acknowledgment

**Comment by clone1018** [comment]
Interesting proposal! Whenever negotiating these types of business deals, it's our responsibility to weigh our time in relation to benefit. If they don't add up, that's where we have to discuss. I've found it helpful in the past to split out the ideas into two sections: 

Benefits for Glimesh: How are we directly improved, rewarded, or otherwise gained by this business partnership. 
Benefits for 3rd Party: How are they directly improved, rewarded, or otherwise gained by this business partnership.  

I do have a couple of direct questions though:

- What is the size of Gamerized? Monthly active users, main usage of the site, etc.
- Is there a demo account, or video we can watch to explore their site? I'm unlikely to sign up an account. 
- Since we are embedding the content, what would the extent of the embedding be? Just the video? Chat? Sub button? Does Gamerized have a subscription model to creators that competes with our primary revenue model?

Anyways, still pondering for more thoughts but thank you for having the conversation and engaging with them!

**Reply by Heiwa1580** [reply]
Sounds good. I can get something worked up that shows an outline of those.

For the questions here is some of the info.

> were at about 23,000 users on the platform with around 3-5k daily active

They are looking into a demo account that can be used. I could also allow anyone who wanted to use my account to use.

When speaking with them the hope would be embedding the video, chat, and sub button. The idea would be that someone could go to their video section and see live streams on the Gamerized platform. The goal for them would be able to offer their customers the ability to have one site to visit for all the content they are looking for. So basically when you go onto their site you basically click on a live stream and while you are still on Gamerized you can still watch and interact with streams as if they were on Gamerized. But it is all powered by Glimesh and run through Glimesh. Users would also need to create Glimesh accounts to chat, and view in the same manner.

![image](https://user-images.githubusercontent.com/68549151/113323516-0c360e00-92e4-11eb-9928-b4e61d4aec6a.png)

![image](https://user-images.githubusercontent.com/68549151/113323623-2f60bd80-92e4-11eb-905f-221131116889.png)

The videos would have a powered by Glimesh statement or some form of attribution and it would be required to use Glimesh in order to have your stream on their site. All subs or future donations (like in platform currency donations) would all still run through us as normal. There would be no additional sub options for subbing to a streamer. The biggest potential conflict is you can choose to donate to a creator on their site from offline videos. But live content would not have that same option. VODs would have that option and those donations go 100% to the streamer Gamerized takes no cut from those donations. I could see this potentially being something that streamers encourage vs subbing to us because Gamerized goes through a paypal donations page, but in the end a streamer could do this without Gamerized.

![image](https://user-images.githubusercontent.com/68549151/113323973-90889100-92e4-11eb-9fac-ca5072b8387c.png)

The only other option that is potentially in conflict with our own revenue model would be their platform subscriptions. But I don't see that affecting us from collecting revenue from subs at all.

![image](https://user-images.githubusercontent.com/68549151/113324208-d80f1d00-92e4-11eb-80b7-e3dfb0718cf9.png)

**Reply by Cykotiq** [reply]
While them hosting VODs sound great, the idea of them embedding live streams on their site feels like it goes too far. As they said, they want to be a centralized platform for all streamers and gamers. It seems to me (and it seems they freely admit this) that this would drive traffic away from our site rather than to it. I don't think an attribution tag is enough to drive significant viewership back to glimesh - we would need to offer something significant that they do not. If viewers can watch their favorite streams and vods on gamerized, why would they visit our site (without vods)? Maybe it's better for us if they **don't** embed chat - if you want to interact with the streamer you need to watch on glimesh - and they could have an "Interact with this streamer on Glimesh" link on the page when they are live.  It could also affect our ability to advertise events if viewers are browsing content on gamerized rather than on glimesh. 

Would they allow us to link to and play the gamerized VODs on the streamer's glimesh channel? We could pull the list of VODs for that streamer via an API, show thumbnails, and embed the VODs on glimesh with attribution. Maybe we only show the last 3 VODs and they click a link to see the rest on gamerized?

their offering of a DMCA free music catalog is also a great value-add with no down side, and I don't see why we can't put them on the https://glimesh.tv/about/credits page

**Comment by Heiwa1580** [comment]
So I'd like to have another meeting with Gamerized. It seems that our major concern is drawing traffic away from Glimesh. 

Did we have anything else we want to go over?

**Reply by clone1018** [reply]
Did you ever get a chance to go over the benefits of both parties? That'll help us make a decision on how we can integrate in a way that helps both of us.

**Reply by Heiwa1580** [reply]
I typed it all up and thought I hit reply on the last message but apparently its not there. I probably never actually hit reply. Let me do that again

**Comment by Heiwa1580** [comment]
Benefits to Glimesh

- Advertising through Gamerized Platform
- Being the official streaming partner of Gamerized (Users if wanting to have streams on Gamerized would be directed to Glimesh)
- Gamerized would host Glimesh VODs (Pros and Cons to this)

Benefits to Gamerized

- They would be able to share live streams on their site and expand their offering (without needing extensive coding)
- Glimesh users would be directed to their platform for VODs
- They would get a host of content added to their offline content through our VODs
- They would have more streamers seeing their DMCA music page

I think this is the general bit of what the benefits would be.

**Reply by clone1018** [reply]
I really like the appeal of being the official streaming partner of Gamerized, I think the question just remains if embedding is right for our current platform. Embedding can be a huge drain on our resources, and it puts video content and our code out into the ether.

**Reply by Heiwa1580** [reply]
Is there an alternative option that we could come up with? Could we like potentially setup a restream option for them where the video is just restreamed through their service?

**Reply by Heiwa1580** [reply]
Hi Clone just wanted to ping you on this one.

**Reply by clone1018** [reply]
You can ping me using @clone1018 else it just shows up as another email. Maybe we should move this brainstorming discussion to something more real time? I have trouble coming up with concrete ideas in isolation.

**Reply by Heiwa1580** [reply]
Cool cool thanks for the heads up. Do we want to perhaps plan a voice chat or just pull it into a Discord channel for discussion? We could perhaps talk to the Gamerized guys about it as well in case they have any thoughts @clone1018

**Comment by Heiwa1580** [comment]
Had another talk with Gamerized.

So just wanted to ask the question if we took out the embedding idea for now and the VODs is there any potential reasons we would not want to partner with Gamerized and work on those aspects over time? 

Basically stick with the cross promotional content and working towards slowly meshing our communities together and over time adding more and more benefits of the partnerships?
